api/parser_handler.go

```diff
@@ -28,25 +28,28 @@ func ParserHandler(w http.ResponseWriter, r *http.Request, mysqlStorage *storage
 	// which he has to use each time even when the claim file error happens
 	err := VerifyCredentialsAndCreateIfNotExists(partnerName, decodedPassword, db)
 	if err != nil {
-		util.WriteError(w, util.AuthError, err.Error())
+		util.WriteMsg(w, err.Error())
+		logrus.Errorf(util.AuthError, err)
 		return
 	}
 
 	// 3. Store claim + claim result into the database
-	if !util.StoreClaimFileInDatabase(db, claimResults, partnerName, ocpVersion, executedBy) {
-		util.WriteError(w, util.ClaimFileError, err.Error())
+	err = util.StoreClaimFileInDatabase(db, claimResults, partnerName, ocpVersion, executedBy)
+	if err != nil {
+		util.WriteMsg(w, err.Error())
+		logrus.Errorf(util.ClaimFileError, err)
 		return
 	}
 
 	// 4. Store file to S3
```

Comment:
The order looks different. Has it been changed? Storing into S3 after storing in the databse?

Reply:
The other PR hasn't been merged yet. so it is based on older version